packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx

```diff
@@ -20,7 +20,7 @@ const MultilineBoxWithRemoveButton = _.compose(
             <div key={`${data[index].id}`} className="flex">
                 <div className="flex-auto">{child}</div>
                 {getRemoveButton({
-                    classes: ['mod-no-border', 'bg-transparent', 'align-end', 'mt1'],
+                    classes: ['mod-no-border', 'align-end', 'mt1'],
```

Comment:
There seem to be a style bug with the disabled remove button.

Before:
![image](https://user-images.githubusercontent.com/35579930/106476531-26cf4080-6475-11eb-87d4-52385bf515fd.png)

After:
![image](https://user-images.githubusercontent.com/35579930/106476490-1d45d880-6475-11eb-97d9-20f1e613c778.png)